cf-keycrypt/cf-keycrypt.c

```diff
@@ -0,0 +1,388 @@
+/*
+   cf-keycrypt.c
+
+   Copyright (C) 2017 cfengineers.net
```

Comment:
A file from external contributors should have the license intact, unless we've made substantial changes. Apache can be relicensed under GPLv3, so this is no problem for the project as a whole